plugins/direct_events_plugin/db/events.php

```diff
@@ -0,0 +1,11 @@
+<?php
```

Comment:
This seems like a dead / unnecessary file. Same thing for the empty `plugins/direct_events_plugin/classes/event/something_happened.php` in this PR.